action.yml

```diff
@@ -118,18 +118,14 @@ runs:
     - name: Set up Release Please
       shell: bash
       run: |
-        git clone https://github.com/stdavis/release-please
-        cd release-please
-        git switch stable-fix
-        echo "installing dependencies"
-        npm install
+        npm i release-please -g
```

Comment:
do we want to add this to a package json or are we ok with not having a static version?

Reply:
Hmm. Good point. There is no package.json file for this project. But I could pin to a major version with the install command. I'll give it a try...